packages/iris-grid/src/IrisGridCopyHandler.tsx

```diff
@@ -252,27 +260,20 @@ class IrisGridCopyHandler extends Component<
     this.setState({ isShown: false });
   }
 
-  copyText(text: string): void {
+  async copyText(text: string): Promise<void> {
     log.debug2('copyText', text);
 
     this.textData = text;
 
-    copyToClipboard(text).then(
-      () => {
-        this.setState({ copyState: IrisGridCopyHandler.COPY_STATES.DONE });
-        this.startHideTimer();
-      },
-      error => {
-        log.error('copyText error', error);
-        this.setState({
-          buttonState: IrisGridCopyHandler.BUTTON_STATES.CLICK_TO_COPY,
-          copyState: IrisGridCopyHandler.COPY_STATES.CLICK_REQUIRED,
-        });
-      }
-    );
```

Comment:
Why was this removed?

Reply:
It was moved down to where the error is caught instead - since we want to show a different error depending on how the copy action occurs (from the initial Copy action that displayed this bar, or from clicking the explicit "Copy" button after that appears).